lib/ez_rubocop/version.rb

```diff
@@ -0,0 +1,3 @@
+module EzRubocop
+  VERSION = "0.49.0".freeze
```

Comment:
Do we want to use the most recent version here?

Reply:
I think we should start with matching what ez-rails is using, so that we can drop this into ez-rails once it has code to share. 

I'd rather burn down some of the existing offenses in ez-rails before introducing another rubocop update there.